test/MUXControlsTestApp/TestInventory.cs

```diff
@@ -12,7 +12,6 @@ static TestInventory()
         {
             Tests = new List<TestDeclaration>();
 
-            Tests.Add(new TestDeclaration("Sample Tests", typeof(SampleTestUIPage)));
```

Comment:
Unrelated to this change as a whole?

Reply:
Yeah, there's a little bit of cleanup like that too. But for me it classified under "remove stuff from the main app project that isn't really common".